infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -128,6 +129,10 @@ private long getStartingSegment() {
      * Create stream log directory if not exists
      */
     private void initStreamLogDirectory() {
+        File corfuDir = new File(logDir.getParent().toString());
+        if (!corfuDir.canWrite()) {
```

Comment:
Do you want to include the path as well?

Reply:
Fixed. Thanks